### ISSUE TITLE
Unclear where to find crawler-cfn.yml

### ISSUE THREAD

**Issue by rddefauw** [awsdocs/aws-cost-management-user-guide#15]
I'm following the [instructions](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/setting-up-athena.html) for setting up Athena to query the cost report.  Under the section _To use the Athena AWS CloudFormation template_, steps 2-3 say:

2. From the list of buckets, choose the bucket image next to the bucket where you chose to receive your AWS Cost and Usage report.
3. Select the template file crawler-cfn.yml and copy the download link to the file.

Those steps are unclear.  The `crawler-cfn.yml` file is actually located in:

`s3://<bucket>/<prefix you specified for the report data>/<name of the report>/crawler-cfn.yml.`

In my case it looked like:

`s3://<my bucket>/cost/athenareport/crawler-cfn.yml`

Basically you have to drill into the folder structure a couple of levels in order to find the file.

**Comment by amiito** [comment]
Thank you for your callout. The steps have been updated here (https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/setting-up-athena.html) to reflect the current console experience. The git hub file will be updated soon to reflect as well. 
Thank you for raising the issue.
